src/agent/src/tracer.rs

```diff
@@ -56,7 +55,7 @@ impl FromStr for TraceType {
     }
```

Comment:
L51-53:
```
"isolated" => Ok(TraceType::Isolated),
"disabled" => Ok(TraceType::Disabled),
_ => Err(TraceTypeError::new("invalid trace type")),
```
Shoud TraceType be removed and replaced with a boolean like it is elsewhere in the code?

Reply:
FYI @jodh-intel on a second round of testing, I got the PR working with no error. However, the code referring to `TraceType` should probably still be removed.